Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -103,10 +104,13 @@ public extension CBPurchase {
     }
 
     //Buy the product
-    func purchaseProduct(product: CBProduct, customerId : String? = "" ,completion handler: @escaping ((_ result: Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)) {
+    func purchaseProduct(product: CBProduct, customerId : String? = "", userInfo : CBCustomerInfo? = nil, completion handler: @escaping ((_ result: Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)) {
```

Comment:
Would suggest to create a new method 
`func purchaseProduct(product: CBProduct, customer: CBCustomer? = nil)`

Reply:
ok we can add in new method aslo. i will add.

Reply:
Here we refactored the existing method to avoid duplication of code , removed customerID and added to CBCustomer and tested details are getting recorded in Test site correctly , can you point or suggest is it mandatory to create new method with same code ?

Reply:
I have created a new function and marked old one as Depreciated